ktor-features/ktor-metrics-micrometer/jvm/src/io/ktor/metrics/micrometer/MicrometerMetrics.kt

```diff
@@ -166,25 +169,52 @@ public class MicrometerMetrics private constructor(
      * Micrometer feature installation object
      */
     public companion object Feature : ApplicationFeature<Application, Configuration, MicrometerMetrics> {
+        @Deprecated("static metrics base name deprecated", level = DeprecationLevel.WARNING)
```

Comment:
no need to deprecate it. it's private, no one will use it anyway. But maybe rename it to `defaultBaseName`?